models/order/src/delivery-item/transformers.ts

```diff
@@ -0,0 +1,21 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TDeliveryItem, TDeliveryItemGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TDeliveryItem, TDeliveryItem>('default', {
+    buildFields: [],
+  }),
+  rest: Transformer<TDeliveryItem, TDeliveryItem>('rest', {
+    buildFields: [],
+  }),
+  graphql: Transformer<TDeliveryItem, TDeliveryItemGraphql>('graphql', {
+    buildFields: [],
+    replaceFields: ({ fields }) => ({
```

Comment:
Since we're not replacing any model's field here, it would be better to use the `addFields` property instead. Doing so will allow us to not need to return the whole fields but just the ones we're adding.